services/rfq/api/db/api_db.go

```diff
@@ -34,6 +38,187 @@ type Quote struct {
 	UpdatedAt time.Time
 }
 
+// ActiveQuoteRequestStatus is the status of a quote request in the db.
+// This is the primary mechanism for moving data through the app.
+//
+// TODO: consider making this an interface and exporting that.
+//
+// EXTREMELY IMPORTANT: DO NOT ADD NEW VALUES TO THIS ENUM UNLESS THEY ARE AT THE END.
+//
+//go:generate go run golang.org/x/tools/cmd/stringer -type=ActiveQuoteRequestStatus
+type ActiveQuoteRequestStatus uint8
+
+const (
+	// Received means the quote request has been received by the server.
+	Received ActiveQuoteRequestStatus = iota + 1
+	// Pending means the quote request is pending awaiting relayer responses.
+	Pending
+	// Expired means the quote request has expired without any valid responses.
+	Expired
+	// Fulfilled means the quote request has been fulfilled.
+	Fulfilled
```

Comment:
nit -- maybe "Closed" instead of "Fulfilled" if we got at least one response? just to avoid any confusion with "Fill" or that any kind of tx occurred yet

Reply:
Agree think Closed is better